src/definitions.ts

```diff
@@ -45,3 +54,23 @@ export interface AdMobInitializationOptions {
    */
   initializeForTesting?: boolean;
 }
+
+export interface AdMobGlobalSettings {
+  /**
+   * If your app has its own volume controls (such as custom music or sound effect volumes),
+   * disclosing app volume to the Google Mobile Ads SDK allows video ads to respect app volume settings.
+   * enable set 0.0 - 1.0
+   *
+   * @see https://developers.google.com/admob/android/global-settings
+   * @since 3.1.0
+   */
+  volume?: number;
+
+  /**
+   * To inform the SDK that the app volume has been muted
+   *
```

Comment:
Maybe we should add this notice here and the `volume` JSDoc from the AdMob docs?

> Note: Video ads that are ineligible to be shown with muted audio are not returned for ad requests made when the app volume is reported as muted or set to a value of 0. This may restrict a subset of the broader video ads pool from serving.

Reply:
This was missing. Thank you for pointing that out!